ItemsCheckerService/Service.cs

```
@@ -0,0 +1,44 @@
+﻿using ItemsCheckerService.Jobs;
+using Quartz;
+using Quartz.Impl;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ItemsCheckerService
+{
+  public class Service
+    {
+        public void Start()
+        {
+            Task.Run(() => UrlsChaker());

```

async void methods will suppress all the exceptions. Avoid them)

---

added logger exception